public/packs/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "admin.css": "/packs/admin-09b1baaa93d7b5b11a46c4c3e14dd5ed.css",
+  "admin.css.map": "/packs/admin-09b1baaa93d7b5b11a46c4c3e14dd5ed.css.map",
```

Comment:
can we remove this file from the commit? pretty sure this shouldnt be checked in